### PR TITLE
Adds npm install chartist --save to readme

### PR DESCRIPTION
**Pull request by debugpai2**
Makes sure #55 doesn't happen again.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ React component for [Chartist.js](https://gionkunz.github.io/chartist-js/)
 ```
 $ npm install react-chartist --save
 ```
+You need to install chartist if you do not have it installed
```
Comment (fraserxu) [inline README.md]: maybe worth note "as a peer dependencie"

Comment (debugpai2) [reply README.md]: I don't follow. Do you mean add it to the README "if you do not have it installed as a peer dependency"?

Comment (fraserxu) [reply README.md]: I mean to point out `chartist` is a peer dependency that has to be install separately in order to use `react-chartist`.

**Comment by debugpai2** [comment]
Made the review changes to point out that chartist is a peer dependency

**Comment by fraserxu** [comment]
Thanks @debugpai2 👍

**Pull request merged**